app/helpers/application_helper.rb

```diff
@@ -1,2 +1,13 @@
 module ApplicationHelper
+
+  #Returns the full title on a per page basis
+  def full_title(page_title = '')
+    base_title = "Organizer Application"
+    if page_title.empty?
```

Comment:
maybe `.blank?` would be a better choice here instead?

Reply:
Oh thank you! I'll change it over. :)